minio_operator_ext/operator.py

```diff
@@ -505,14 +503,7 @@ async def create_minio_group_binding(
 
             def inner():
                 # NOTE: ensure group already exists before calling 'group_add' to modify members
-                try:
-                    minio_admin_client.group_info(group_binding.group)
-                except minio.error.MinioAdminException as e:
-                    if e._code != "404":
-                        raise e
-                    raise operator_core.OperatorError(
-                        f"group does not exist: {group_binding.group}", recoverable=True
-                    )
+                minio_admin_client.group_info(group_binding.group)
```

Comment:
Both code paths return a recoverable exception - and there's no 'improvement' to readability in wrapping the existing exception with an `OperatorError` (- the base minio exception also indicates that the group does not exist).  

Consequently, just remove the try/except altogether.